### PR TITLE
Puppet 4 considers empty string boolean true

### PR DESCRIPTION
**Pull request by jtreminio**

### CODE REVIEW

File: manifests/netinstall.pp [hunk 1]
```diff
@@ -113,7 +113,7 @@
     default => $postextract_cwd,
   }
 
-  if $preextract_command {
+  if $preextract_command != '' {
```
Comment (alvagante) [inline manifests/netinstall.pp]: I'd check here also for false values so:
if $preextract_command and $preextract_command != ''

File: manifests/netinstall.pp [hunk 2]
```diff
@@ -153,7 +153,7 @@
     environment => $exec_env,
   }
 
-  if $postextract_command {
+  if $postextract_command != '' {
```
Comment (alvagante) [inline manifests/netinstall.pp]: same here

**Comment by jtreminio** [comment]
@alvagante done and squashed.

**Comment by alvagante** [comment]
+1

**Pull request merged**